src/main/java/no/sikt/nva/BrageMigrationCommand.java

```diff
@@ -189,16 +196,14 @@ public Integer call() {
                 var contributors = getContributors(inputDirectory);
                 var affiliations = AffiliationsScraper.getAffiliations(new File(inputDirectory + AFFILIATIONS_FILE));
                 printIgnoredDcValuesFieldsInInfoLog();
+                onlineEmbargoChecker.calculateCustomerAddress(customer);
+                onlineEmbargoChecker.setOutputDirectory(outputDirectory);
```

Comment:
Because we only know the customer at this point, I can make onlineEmbargoChecker take the customer and output directory in its constructor as I want to inject the onlinEmbargoChecker during testing.